src/MOIWrapper.jl

```diff
@@ -374,3 +374,40 @@ end
 function LQOI.delete_variables!(instance::Optimizer, start_col::Int, end_col::Int)
     delete_columns(instance.inner, [Cint(i-1) for i in start_col:end_col])
 end
+
+const statmap = Dict(zip([ 0x00,  0x01, 0x02, 0x03, 0x04, 0x05],
```

Comment:
Please link to the Clp source code where these constants are defined.

Reply:
https://github.com/coin-or/Clp/blob/master/Clp/src/Clp_C_Interface.h
```
/* status values are as in ClpSimplex.hpp i.e. 0 - free, 1 basic, 2 at upper,
        3 at lower, 4 superbasic, (5 fixed) */
/* Get variable basis info */
COINLIBAPI int COINLINKAGE Clp_getColumnStatus(Clp_Simplex *model, int sequence);
```
should we include this in the MOIWrapper.jl code?
(I admit that the `Dict(zip(` ... is a copy from MathProgBase wrapper.

Reply:
The original definition from https://github.com/coin-or/Clp/blob/master/Clp/src/ClpSimplex.hpp : 
```
  enum Status {
    isFree = 0x00,
    basic = 0x01,
    atUpperBound = 0x02,
    atLowerBound = 0x03,
    superBasic = 0x04,
    isFixed = 0x05
};
```

Reply:
> should we include this in the MOIWrapper.jl code?

Yes, add a comment to the code explaining where the constants are defined, i.e., in `ClpSimplex.hpp`.